procedures/auxtel/observation_procedures/Parabolic_Focus_Determination_Data_Acq.ipynb

```diff
@@ -9,23 +9,42 @@
     "\n",
```

Comment:
<p>Consider:</p><h2>Mirror Alignment and focus without a filter and grating</h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
<p>Already fix by you</p>